tests/testcases/decl-module-or-global/index.ts

```diff
@@ -0,0 +1,10 @@
+declare module "babel__core" {
+  var fn1: any
```

Comment:
Like I said above, it might be good to actually reference a local type here, so that we make sure that it is correctly pulled in.
For example `const export1: A;` here, and somewhere else just define an empty `interface A {}`